WordPress/Classes/Services/AppUpdate/AppStoreSearchService.swift

```diff
@@ -10,6 +10,13 @@ struct AppStoreLookupResponse: Decodable {
         let version: String
         let releaseNotes: String
         let minimumOsVersion: String
+        let currentVersionReleaseDate: Date
+
+        func currentVersionHasBeenReleased(for days: Int) -> Bool {
+            let secondsInDay: TimeInterval = 86_400
+            let secondsSinceRelease = -currentVersionReleaseDate.timeIntervalSinceNow
+            return secondsSinceRelease > Double(days) * secondsInDay
```

Comment:
(nit) there is a convenience API for this:

```
guard let days = Calendar.current.dateComponents([.day], from: lhs, to: rhs).day else {
            return wpAssertionFailure("should-never-happen")
}
return days > 1
```

**warning:** I have not tested it!

Reply:
I've addressed this in https://github.com/wordpress-mobile/WordPress-iOS/pull/23221 🙇‍♀️